pkg/kubecli/vm.go

```diff
@@ -35,14 +35,15 @@ import (
 
 type KubevirtClient interface {
 	VM(namespace string) VMInterface
+	Migration(namespace string) MigrationInterface
 }
 
 type kubevirt struct {
 	restClient *rest.RESTClient
 }
 
 func (k *kubevirt) VM(namespace string) VMInterface {
-	return &vms{k.restClient, namespace}
+	return &vms{k.restClient, namespace, "vms"}
```

Comment:
This isn't really related to the patch, but it seems like we should define the literal "vms" at some point.